packages/zipkin-instrumentation-grpc/src/ZipkinGrpcInterceptor.js

```diff
@@ -61,10 +61,8 @@ class ZipkinGrpcInterceptor {
       return;
     }
 
-    let ctxSampled = grpcMetadataFromIncomingCtx.get('x-b3-sampled')[0];
-    if (!(ctxSampled in ['0', '1'])) {
-      ctxSampled = '0';
-    }
+    let ctxSampledAsString = grpcMetadataFromIncomingCtx.get('x-b3-sampled')[0];
```

Comment:
cool. while unlikely, the header could be not present (that's why the Option type is used)

Reply:
The grpcMetadataFromIncomingCtx is a grpc.Metadata type.
If the header is not present it returns an empty array []; Taking the zeroth value will then be an undefined.
An undefined is not equal to either '1' or 'true' and thus result to false.
So, this should be fine, right?